wfdb/io/convert/edf.py

```diff
@@ -399,7 +399,7 @@ def read_edf(
         }
 
     sig_data = np.empty((sig_len, n_sig))
-    temp_sig_data = np.fromfile(edf_file, dtype=np.int16)
+    temp_sig_data = np.fromfile(edf_file, dtype=np.int16).astype("int64")
```

Comment:
edf.py was updated in https://github.com/MIT-LCP/wfdb-python/pull/527 so this change is not needed.

Reply:
Got it!